app/Judite/Models/Enrollment.php

```diff
@@ -94,9 +94,8 @@ public function exchange(self $enrollment): self
         $fromShiftId = $this->shift_id;
         $this->shift()->associate($enrollment->shift_id);
         $enrollment->shift()->associate($fromShiftId);
-
+		$enrollment->save();
         $this->save();
```

Comment:
```suggestion
		
        $this->save();
        $enrollment->save();
```

Why did you have to change this? Not saying it is wrong, just curious as to why